mock/py/mockbuild/buildroot.py

```diff
@@ -283,7 +283,7 @@ def _fallback(message):
                 if digest_expected:
                     getLog().info("Checking image digest: %s",
                                   digest_expected)
-                    digest = podman.get_image_digest()
+                    digest = podman.get_layers_digest()
                     if digest != digest_expected:
                         getLog().warning(
```

Comment:
Do we want to make this failing again?

Reply:
Probably needs a bit of testing in the wild. I would turn it on by default e.g. one or two version later.